python/src/vmaf/core/quality_runner.py

```diff
@@ -970,6 +972,7 @@ def predict_with_model(cls, model, xs, **kwargs):
                 'ys_pred_stddev': ys_pred_stddev,
                 'ys_pred_ci95_low': ys_pred_ci95_low,
                 'ys_pred_ci95_high': ys_pred_ci95_high,
+                'num_type_bootstrap_models': num_type_bootstrap_models,
```

Comment:
Do you really need to pass this out? Can't this be figured out by the length of the list of 'all_models'?

Reply:
Yes, new PR does that.